target/linux/ar71xx/image/generic.mk

```diff
@@ -722,6 +722,15 @@ define Device/xd3200
 endef
 TARGET_DEVICES += xd3200
 
+define Device/xd6700
+  DEVICE_TITLE := YunCore XD6700
+  DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca9887
+  BOARDNAME = XD6700
+  CONSOLE = ttyS0,115200
+  MTDPARTS := spi0.0:256k(u-boot)ro,64k(u-boot-env),16000k(firmware),64k(art)ro
```

Comment:
If you don't use/access `u-boot-env` partition in user space and don't make it usable with `uboot-envtools` package, mark it please as `ro`.